### PR TITLE
Add `--path .` to cargo install commands.

### PR DESCRIPTION
**Pull request by lbeckman314**
Changing `cargo install` to `cargo install --path .` prevents the following error:

**error:** Using `cargo install` to install the binaries for the package in current working directory is no longer supported, use `cargo install --path .` instead. Use `cargo build` if you want to simply build the package.

### CODE REVIEW

File: book-example/src/for_developers/backends.md [hunk 1]
```diff
@@ -171,6 +171,16 @@ Now imagine you don't want to count the number of words on a particular chapter
...
 
+```diff
+  [output.wordcount]
++ ignores = [
++     "For Developers",
++     "Preprocessors",
++     "Alternative Backends",
++ ]
++ deny-odds = true
+```
```
Comment (steveklabnik) [inline book-example/src/for_developers/backends.md]: this change seems unrelated, was it intended?

Comment (lbeckman314) [reply book-example/src/for_developers/backends.md]: Ah, sorry! You're right — that was an unrelated commit. I have reset to the earlier commit. Thanks!
